### ISSUE TITLE
Stellar-Polygon Bridge

### ISSUE THREAD

**Issue by robvanmieghem** [threefoldfoundation/tft#165]
Requirements:
- [ ] A publicly available instance of https://github.com/threefoldtech/libp2p-relay

TODO:

- [ ]  #164

**Comment by amiraabouhadid** [comment]
I think we already know if a bor node can run standalone, no? 

![Screenshot from 2022-10-20 11-18-53](https://user-images.githubusercontent.com/56790126/197723349-db09e5fa-ea57-42a4-bd71-c551963c9420.png)

**Comment by robvanmieghem** [comment]
yes, but we normally run light clients in the bridge, not full nodes, How big is the storage?
Can we also move this discussion to issue #164 and not on this high level one?

**Comment by amiraabouhadid** [comment]
working on polygon bridge frontend
